src/App.jsx

```diff
@@ -1,18 +1,24 @@
 import { render } from "react-dom";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import Home from "./pages/Home";
-import Singles from "./pages/Singles";
-import Albums from "./pages/Albums";
-import PlayLists from "./pages/PlayLists";
 
 const App = () => {
   return (
     <BrowserRouter>
       <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/singles" element={<Singles />} />
-        <Route path="/albums" element={<Albums />} />
-        <Route path="/playlists" element={<PlayLists />} />
+        <Route path="/" element={<Home title="New albums and singles" />} />
```

Comment:
- Please use children prop.
- And change the name to `MainLayout`.
Something like:
```jsx
<MainLayout>
  <Singles />
</MainLayout>
```here

Reply:
oooooh such a great idea.
I did it.
Thanks